hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCallSuite.java

```diff
@@ -1569,6 +1572,43 @@ HapiSpec invalidContract() {
                 .then(contractCallWithFunctionAbi("invalid", function).hasKnownStatus(INVALID_CONTRACT_ID));
     }
 
+    // This test for modularization
+    HapiSpec invalidContractDoesNotExist() {
+        final AtomicReference<AccountID> accountID = new AtomicReference<>();
+        final var function = getABIFor(FUNCTION, "getIndirect", CREATE_TRIVIAL);
+
+        return defaultHapiSpec("invalidContractDoesNotExist")
+                .given(cryptoCreate("invalid")
```

Comment:
Could remove the `cryptoCreate()` here for clarity.

Reply:
yes, makes sense.  removed.